@here/harp-datasource-protocol/lib/Techniques.ts

```diff
@@ -342,6 +343,8 @@ export function textureCoordinateType(technique: Technique): TextureCoordinateTy
         return technique.textureCoordinateType;
     } else if (isTerrainTechnique(technique)) {
         return technique.textureCoordinateType;
+    } else if (isFillTechnique(technique)) {
+        return TextureCoordinateType.TileSpace;
```

Comment:
Is there a reason this is different from the others, i.e. all the others mention `technique.textureCoordinateType`

Reply:
The difference with the other techniques listed there is that FillTechnique does not expose any texture parameter, so it doesn't make sense to have a textureCoordinateType parameter either.
However we do need to generate texture coordinates for the elevation displacement map.

Anyway, I'll remove the line you commented, it's not needed because we take care of generating texture coordinates for the fill technique in OmvDecodedTileEmitter.